src/api/app/views/webui/package/_basic_info.html.haml

```diff
@@ -8,3 +11,20 @@
       %i No description set
     - else
       = render partial: 'webui/shared/collapsible_text', locals: { text: package.description }
+
+  - if policy(package).update?
+    = link_to('javascript:void(0);', id: 'toggle-in-place-editing', class: 'nav-link', title: 'Edit') do
+      %i.fas.fa-edit
+      %span.nav-item-name Edit
+
+:javascript
+  $('#toggle-in-place-editing').on('click', function () {
+    $('.in-place-editing .basic-info').toggle();
+    $('.in-place-editing .editing-form').toggle();
+    return false;
```

Comment:
What is the reason for returning `false`?

Reply:
None, it's something left forgotten from using buttons instead of links. I'll remove it